src/routes/main.js

```diff
@@ -0,0 +1,12 @@
+module.exports = [
+  {
+    path: '/',
+    method: 'GET',
+    handler: (request, reply) => {
+      if (!request.auth.isAuthenticated) {
```

Comment:
As you have now set the default auth mode to required (see below), you need to explicitly set it to `try` here